### PR TITLE
Feat/request GitHub issues

### PR DESCRIPTION
**Pull request by shendriksen**

### CODE REVIEW

File: src/actions/githubActions.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-        dispatch({
-            type: GITHUB_FETCH_ISSUES,
-            payload: {
-                timescomponents: timesComponentIssues,
-                dextrose: dextroseIssues
-            }
-        });
+
+        Promise
+            .all(requestBuilder.buildIssueRequests(repositories))
```
Comment (L0wry) [inline src/actions/githubActions.js]: lint will want this to be destructured

File: src/actions/githubActions.js [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+                return results.reduce((object, currentValue) => {
+                    return Object.assign(object, {
+                        [currentValue.project]: currentValue.issues
+                    });
+                }, {});
+            })
+            .then(payload => dispatch({type: GITHUB_FETCH_ISSUES, payload}))
+            .catch((error) => dispatch({type: GITHUB_FETCH_ISSUES, payload: error, error: true}));
     };
+
```
Comment (L0wry) [inline src/actions/githubActions.js]: spacey wacey

File: src/components/GithubIssues.js [hunk 3]
```diff
@@ -3,11 +3,12 @@ import {bindActionCreators} from 'redux';
 import * as githubActions from '../actions/githubActions';
 import PropTypes from 'prop-types';
 import React from 'react';
+import TextComponent from './TextComponent';
 
 const renderData = (item) => {
     return Object.keys(item).map(value => {
         const row = `${value} has ${item[value].issueCount} thing or things`
```
Comment (L0wry) [inline src/components/GithubIssues.js]: we can remove row now

File: src/actions/githubActions.js [hunk 4]
```diff
@@ -1,16 +1,21 @@
-import timesComponentIssues from  './../data/mockgithubissuesTimesComponent.json';
-import dextroseIssues from  './../data/mockgithubissuesDextrose.json';
+import repositories from '../config/repositories';
+import requestBuilder from '../helpers/requestBuilder';
 
 export const GITHUB_FETCH_ISSUES = 'GITHUB_ISSUES_FETCH';
```
Comment (asifhafeez) [inline src/actions/githubActions.js]: Naming inconsistency can lead to confusion, these should be the same.

File: src/components/TextComponent.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const TextComponent = ({children, size}) => {
     const styles = {
         color: calculateColour(size),
-        fontSize: `${size}px`
+        fontSize: `${size || 10}px`
```
Comment (asifhafeez) [inline src/components/TextComponent.js]: Can you make '10' a constant?

File: src/actions/githubActions.js (hunk 4 above)
Comment (L0wry) [reply src/actions/githubActions.js]: que?

**Pull request merged**